### PR TITLE
Add HOST command

### PR DESCRIPTION
**Pull request by dchw**
Implements the feature discussed in #1168. Pretty simple, direct mapping with no frills.

Since `/etc/hosts` tends to be controlled when launching built containers; these may not persist or be accessible once the image is built. However, they will persist for the duration of your build.

Example:

```
add-single-host:
    FROM alpine:3.15
    HOST example.com 1.2.3.4
    RUN wget example.com
```

### CODE REVIEW

File: features/features.go [hunk 1]
```diff
@@ -27,6 +27,7 @@ type Features struct {
 	EarthlyVersionArg          bool `long:"earthly-version-arg" description:"includes EARTHLY_VERSION and EARTHLY_BUILD_SHA ARGs"`
 	ExplicitGlobal             bool `long:"explicit-global" description:"require base target args to have explicit settings to be considered global args"`
 	UseCacheCommand            bool `long:"use-cache-command" description:"allow use of CACHE command in Earthfiles"`
+	UseHostCommand             bool `long:"use-host-command" description:"allow use of HOST command in Earthfiles"`
```
Comment (vladaionescu) [inline features/features.go:30]: Mind adding this to 0.7? It's lower in this file somewhere.

**Review by vladaionescu — APPROVED** [review]
Very cool 🎉

**Comment by vladaionescu** [comment]
Oh - mind adding an item in the changelog? We should mention how users can enable this experimental feature.

**Comment by dchw** [comment]
Can do!

**Pull request merged**